### DISCUSSION TITLE
Nicholas Norwood

### DISCUSSION THREAD

**Discussion by TokyoTurtle22** [birkenkrahe/ai482#13] [Show and tell]
https://www.youtube.com/watch?v=d3XgIP6L_lw&t=139s

**Comment by birkenkrahe** [comment]
Enjoyed this very much. One of my best friends is a former National German chess champion, and that's how close I've ever come to success in chess. I found the plots at around 5 min of the talk interesting. There's a lot one can do with the scatterplot in particular, to identify patterns...e.g. with R. The TopCoder adversarial agent was fascinating, and the whole matchmaking issue inspired me to look more deeply into it, so you have created a new disciple! (I've done a lot of work in gamification, so I've got some foundation). Cheers, MB
